### ISSUE TITLE
New term request: Autoimmune pancreatitis

### ISSUE THREAD

**Issue by yankeetomer** [obophenotype/human-phenotype-ontology#5817]
**Preferred term label:**
Autoimmune pancreatitis

**Synonyms**
Lymphoplasmacytic Sclerosing Pancreatitis

PMID: 22666688

>  AIP is also known by other names including lymphoplasmacytic sclerosing pancreatitis with cholangitis, idiopathic duct destructive pancreatitis, primary inflammatory pancreatitis, non-alcoholic duct destructive chronic pancreatitis, tumefactive pancreatitis, and destructive pancreatitis depending on the specific tissue changes found on biopsy or the predominant and accompanying symptoms.

**Definition (free text, please give PubMed ID)**
A rare form of chronic pancreatitis associated with autoimmune manifestations.

PMID: 24891978
PMID: 22666688

> Autoimmune pancreatitis (AIP) was fist described by Yoshida et al[1] in 1995, and introduced to be a form of chronic pancreatitis associated with autoimmune manifestations. Today it has been known that the disease represents a systemic autoimmune condition that involves not only the pancreas but also a variety of other organs such as bile duct[2], the retroperitoneum[3], and lymph nodes[4].

**Parent term (use hpo.jax.org/app)**
Pancreatitis 
HP:0001733
https://hpo.jax.org/app/browse/term/HP:0001733

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://hpo.jax.org/app/browse/term/HP:0001733

**Your nano-attribution (ORCID)**
Tomer Talmy M.D.
Organization: Emedgene Technologies
ORCID: https://orcid.org/0000-0002-7440-8864

**Comment by pnrobinson** [comment]
@nicolevasilevsky  I think that this item should actually be a Mondo term and not an HPO term. @yankeetomer this seems to be a disease rather than a phenotypic feature. We classify diseases using the Mondo ontology (https://www.ebi.ac.uk/ols/ontologies/mondo).
@nicolevasilevsky please close this item if you agree.

**Comment by nicolevasilevsky** [comment]
I agree.
This term is already in Mondo:
MONDO_0015175 'autoimmune pancreatitis'
